macros/edr/data_monitoring/anomaly_detection/get_anomaly_scores_query.sql

```diff
@@ -111,7 +111,7 @@
         anomaly_scores as (
 
             select
-                {{ dbt_utils.surrogate_key([
+                {{ dbt_utils.generate_surrogate_key([
```

Comment:
Older versions of dbt-utils don't have this macro, so we need to create a macro that is called generate_surrogate_key (similar to what we did [here](https://github.com/elementary-data/dbt-data-reliability/blob/master/macros/utils/cross_db_utils/current_timestamp.sql)).
Currently there is an issue with getting the installed version of dbt-utils so as a start lets do the following check:
Checks the dbt version, and if it is lower than 1.3.0 it calls `dbt_utils.surrogate_key` and if its equal or bigger it calls `dbt_utils.generate_surrogate_key`

(We will ask in our guide to set dbt utils package to be higher than 1.0.0 if working with dbt 1.3.0)